js/src/dom/selectorEngine.js

```diff
@@ -46,7 +46,7 @@ const SelectorEngine = {
     return fnMatches.call(element, selector)
   },
 
-  find(selector) {
+  find(element = document, selector) {
```

Comment:
Nice change 👍 I did the same but without the default value and your change is much better 😄